src/SFML/Window/DRM/DRMContext.cpp

```diff
@@ -573,28 +578,27 @@ m_scanOut    (false)
     createContext(shared);
 
     if (owner)
-    {
-        Vector2u size = owner->getSize();
-        createSurface(size.x, size.y, bitsPerPixel, true);
-    }
+        createSurface(true);
 }
 
 
 ////////////////////////////////////////////////////////////
-DRMContext::DRMContext(DRMContext* shared, const ContextSettings& settings, unsigned int width, unsigned int height) :
+DRMContext::DRMContext(DRMContext* shared, const ContextSettings& settings, unsigned int /*width*/, unsigned int /*height*/) :
```

Comment:
Why comment out these parameters instead of just removing them?

Reply:
The parameters need to stay as this constructor is used in Window/GlContext.cpp